.puppeteerrc.cjs

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+const os = require("os");
+
+/**
+ * @see https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#could-not-find-expected-browser-locally
+ * @type {import("puppeteer").Configuration}
+ */
+module.exports = {
+  cacheDirectory: path.join(os.homedir(), '.cache', 'puppeteer')
```

Comment:
Will chromedriver already download (and cache) a copy in `/usr/bin/google-chrome-stable`?

The PR is out of date but I had a quick look here:

[fix(ci): update chromedriver version](https://github.com/dequelabs/axe-core-npm/pull/652/commits/5b40122ee0e9e0f59b4e76e2a845fae224bf0cb1)
[fix(ci): add puppeteer executable path](https://github.com/dequelabs/axe-core-npm/pull/652/commits/d839660d9a56e187f820842459d00d958fe89be1)

Reply:
Thanks. Your changes worked so I cherry-picked them in to this pr.